src/psyclone/tests/test_files/dynamo0p3/1.6_single_invoke_2_int_scalars.f90

```diff
@@ -14,12 +14,10 @@ program single_invoke
   use inf,      only: field_type
   implicit none
   type(field_type) :: f1, f2, m1, m2
-  real(r_def)      :: a
-  integer(i_def)   :: iflag, istep
+  integer(i_def)   :: iflag, istep, iscalar
 
   call invoke(                                  &
        testkern_type(iflag,f1,f2,m1,m2,istep),  &
-       testkern_type(1_i_def,f1,f2,m1,m2,istep) &
-          )
+       testkern_type(iflag,f1,f2,m1,m2,iscalar)   )
```

Comment:
This test was explicitly checking the case where an argument is passed by value. Please could you revert this part of the change (and the associated pytest test of course).

Reply:
This one is a bit harder @arporter. PSyclone doesn't generate the PSy layer with `i_def`. Not in the module use statement, nor in any of the declarations. The generated algorithm code doesn't have the literal constant passed to the PSy layer. It only appears as an argument to the kernel call in the PSy layer. So, I can't have a test for when an argument is passed in as a literal constant, because it PSyclone doesn't generate the correct code.